### ISSUE TITLE
VZV | Map | Add spinner when map data is fetching

### ISSUE THREAD

**Issue by mddilley** [cityofaustin/atd-data-tech#1293]
When data is being fetched for the when filtering crash data or applying overlays, it is hard to see what is happening from the user perspective. Need to add a spinner of loading message to show that data is coming.

@JaceDeloney Creating this as a placeholder, feel free to add any thoughts you have on how this should look in the viewer.

**Comment by JaceDeloney** [comment]
This is a good idea. I grouped it with the other design and ux polish issues (currently in Sprint 18, might eventually move to 19). It would be cool to have some sort of compass loading animation a la the top right icon here
https://www.pinterest.com/pin/453878468694056448/

I'm not entirely sure what the realm of possibility is. Do we have to use something out of the box from mapbox?

**Comment by mddilley** [comment]
Yep, @sergiogcx brought up this idea this morning when he was reviewing the overlays, and I agree 100%.

I found this example with a quick search. We could customize the amount of spin/waggle too.
https://codepen.io/wavyknife/pen/JdEGQR
Or apply similar animation to:
https://fontawesome.com/icons/compass?style=regular

**Comment by JaceDeloney** [comment]
![emoji_emoticon_face_mood-03-512.png](https://images.zenhubusercontent.com/5dc05535495c1e0001182c16/c959ec19-01f7-407a-b3ab-28a6e345d3f4)

**Comment by JaceDeloney** [comment]
Moving this to atd-vz-data
